### PR TITLE
Update CRX known issues page

### PR DESCRIPTION
**Pull request by sebastianbenz**

### CODE REVIEW

File: site/en/docs/extensions/migrating/known-issues/index.md [hunk 1]
```diff
@@ -15,69 +15,48 @@ tags:
 
 {% endAside %}
 
-This page lists major known issues that affect developers as they migrate to Manifest V3. Known issues are divided into two primary groups:
+Recently, we announced changes to the Manifest V2 deprecation timeline, and while we remain firmly committed to Manifest V3 we acknowledge there is more work to do on our part including:
```
Comment (rdcronin) [inline site/en/docs/extensions/migrating/known-issues/index.md:18]:
```suggestion
Recently, we announced changes to the Manifest V2 (MV2) deprecation timeline, and while we remain firmly committed to Manifest V3 (MV3) we acknowledge there is more work to do on our part including: 
```

File: site/en/docs/extensions/migrating/known-issues/index.md [hunk 2]
```diff
@@ -15,69 +15,48 @@ tags:
...
-These lists are a curated subset of items from the [Chromium issue tracker][crbug]. For a complete list of issues, visit the issue tracker and search for items tagged with the [Platform>Extensions][crbug-crx] component.
+## Closing the Platform Gap
 
-When a capability is added to the platform or a bug is addressed, it will be removed from this page at roughly the same time the change lands in Chrome's Stable channel. See [What's New in Chrome Extensions][crx-whats-new] for a list of recent updates.
+We are committed to closing the following gaps before announcing a new MV2 deprecation timeline:
 
-## Capabilities {: #capabilities }
+1. **User Script support:** Allow registering content scripts with arbitrary code by adding new functionality to the scripting API (see [our proposal](https://github.com/w3c/webextensions/blob/main/proposals/user-scripts-api.md) for more detail).
+1. Further **Service Worker lifecycle enhancement:** 
+    * Additional strong Service Worker keepalives for operations taking longer than 5 minutes.
```
Comment (rdcronin) [inline site/en/docs/extensions/migrating/known-issues/index.md:29]:
```suggestion
    * Additional strong Service Worker keepalives for certain operations taking longer than 5 minutes.
```

Comment (rdcronin) [inline site/en/docs/extensions/migrating/known-issues/index.md:29]:
```suggestion
    * Additional strong Service Worker keepalives for certain operations taking longer than 5 minutes.
```

File: site/en/docs/extensions/migrating/known-issues/index.md [hunk 3]
```diff
@@ -15,69 +15,48 @@ tags:
...
+    * Extend service worker lifetimes while there are active WebSocket connections.
+1. **Increase the number of static and enabled rulesets** for Declarative Net Request (DNR).
+1. Extend **[Offscreen document](/docs/extensions/reference/offscreen/) functionality** to support more reasons for using an Offscreen document.
+1. Improving **support for the `chrome.tabCapture` API**:
```
Comment (rdcronin) [inline site/en/docs/extensions/migrating/known-issues/index.md:33]:
```suggestion
1. Improving **support for the [`chrome.tabCapture`](https://developer.chrome.com/docs/extensions/reference/tabCapture/) API**:
```

File: site/en/docs/extensions/migrating/known-issues/index.md [hunk 4]
```diff
@@ -15,69 +15,48 @@ tags:
...
+    * Support calling `getMediaStreamId` from a service worker.
+    * Support obtaining a MediaStream from a stream ID in an offscreen document.
+1. **Support File Handling API on ChromeOS** as a replacement for `chrome.fileBrowserHandler`.
```
Comment (rdcronin) [inline site/en/docs/extensions/migrating/known-issues/index.md:36]:
```suggestion
1. **Support File Handling API on ChromeOS** as a replacement for [`chrome.fileBrowserHandler`](https://developer.chrome.com/docs/extensions/reference/fileBrowserHandler/).
```

File: site/en/docs/extensions/whatsnew/index.md [hunk 5]
```diff
@@ -16,6 +16,12 @@ tags:
 Check this page often to learn about changes to the Chrome extensions platform, its documentation,
 and related policy or other changes. You'll find other notices posted on the [Extensions Google Group](https://groups.google.com/a/chromium.org/g/chromium-extensions). The [Extensions News](/tags/extensions-news/) tag lists articles about many of the topics listed here. (It even has [an RSS feed](/feeds/extensions-news.xml).)
 
+### Updated MV3 migration guidance 
+
+<p class="color-secondary-text type--caption">Posted on <time>May 9, 2023</time></p>
+
+We've updated the [know issues section of our MV3 migration guidance](/docs/extensions/migrating/known-issues/), with a updated list of extension platform gaps that we intent to close before announcing a new MV2 deprecation timeline.
```
Comment (rdcronin) [inline site/en/docs/extensions/whatsnew/index.md:23]:
```suggestion
We've updated the [known issues section of our MV3 migration guidance](/docs/extensions/migrating/known-issues/) with a updated list of extension platform gaps that we intend to close before announcing a new MV2 deprecation timeline.
```

File: site/en/docs/extensions/migrating/known-issues/index.md [hunk 6]
```diff
@@ -15,69 +15,48 @@ tags:
 
 {% endAside %}
 
-This page lists major known issues that affect developers as they migrate to Manifest V3. Known issues are divided into two primary groups:
+Recently, we announced changes to the Manifest V2 (MV2) deprecation timeline, and while we remain firmly committed to Manifest V3 (MV3) we acknowledge there is more work to do on our part including:
```
Comment (rdcronin) [inline site/en/docs/extensions/migrating/known-issues/index.md:18]:
```suggestion
Recently, we announced changes to the Manifest V2 (MV2) deprecation timeline, and while we remain firmly committed to Manifest V3 (MV3) we acknowledge there is more work to do on our part.
```

Optional, but the "including" followed by bullets for "we will do <x>" sounds a little awkward.

File: site/en/docs/extensions/migrating/known-issues/index.md [hunk 7]
```diff
@@ -15,69 +15,48 @@ tags:
 
 {% endAside %}
 
-This page lists major known issues that affect developers as they migrate to Manifest V3. Known issues are divided into two primary groups:
+Recently, we announced changes to the Manifest V2 (MV2) deprecation timeline, and while we remain firmly committed to Manifest V3 (MV3) we acknowledge there is more work to do on our part including: 
 
-* **Capabilities** – Features that we plan to add to Manifest V3 to facilitate migration efforts.
-* **Bugs** – Significant issues with Manifest V3 platform features that are not working as expected.
+*   Before announcing a new timeline for deprecation, we will finish addressing prioritized platform gaps and closing critical bugs that are documented on this page.
+*   We will give developers time to build, guaranteeing at least 6 months between a timeline announcement and any experiments deprecating MV2 features.
```
Comment (rdcronin) [inline site/en/docs/extensions/migrating/known-issues/index.md:21]:
```suggestion
*   We will give developers time to build, guaranteeing at least 6 months between a timeline announcement and any experiments removing support for MV2.
```

I feel like this is a bit more accurate (we may _separately_ deprecate features as part of normal platform maintenance)

**Review by rdcronin — APPROVED** [review]
LGTM; thanks, Sebastian!

File: site/en/docs/extensions/whatsnew/index.md [hunk 8]
```diff
@@ -16,6 +16,12 @@ tags:
 Check this page often to learn about changes to the Chrome extensions platform, its documentation,
 and related policy or other changes. You'll find other notices posted on the [Extensions Google Group](https://groups.google.com/a/chromium.org/g/chromium-extensions). The [Extensions News](/tags/extensions-news/) tag lists articles about many of the topics listed here. (It even has [an RSS feed](/feeds/extensions-news.xml).)
 
+### Updated MV3 migration guidance 
+
+<p class="color-secondary-text type--caption">Posted on <time>May 9, 2023</time></p>
+
+We've updated the [known issues section of our MV3 migration guidance](/docs/extensions/migrating/known-issues/) with an updated list of extension platform gaps that we intend to close before announcing a new MV2 deprecation timeline.
```
Comment (jpmedley) [inline site/en/docs/extensions/whatsnew/index.md:23]:
```suggestion
We've updated the [Known Issues](/docs/extensions/migrating/known-issues/) section of our [Manifest V3 migration guidance](/docs/extensions/migrating/) with an updated list of extension platform gaps that we intend to close before announcing a new Manifest V2 deprecation timeline.
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

**Review by jpmedley — APPROVED** [review]

**Pull request merged**